Fix #43 - Adjusted font size and layout of sample page

saaxena opened the pull request:
following are the changes which i made  - 

Consistent Spacing: Added proper padding and margins throughout components

Typography Hierarchy: Adjusted font sizes and weights for better visual hierarchy

Enhanced Visuals: Added subtle shadows, hover effects, and transitions

Responsive Layout: Improved mobile behavior with appropriate breakpoints

Visual Separation: Added proper spacing between components

Interactive Elements: Enhanced buttons and interactive elements with hover states

Improved Container Structure: Used fluid containers and proper grid classes


The changes Maintain the standalone and independent functionality of each component while giving a more better look . 

After and Before  Visuals comparison  - 
####After  - 
![Screenshot 2025-03-19 165350](https://github.com/user-attachments/assets/3c8e3e91-1df9-4a0b-9d95-d7dd20d6c16d)
![Screenshot 2025-03-19 165411](https://github.com/user-attachments/assets/ad38ca7e-4b7f-43ee-9b2f-936263bdbbb9)

##### Before  - 
![Screenshot 2025-03-19 182118](https://github.com/user-attachments/assets/df690daa-3930-4328-878a-e0fd95a36972)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/RocketChat/GSoC-Community-Hub?pullRequest=48) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/RocketChat/GSoC-Community-Hub?pullRequest=48) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RocketChat/GSoC-Community-Hub?pullRequest=48) it.</sub>

Sing-Li commented on sample.env.local:
```diff
@@ -1 +0,0 @@
-
```
I don't think this files should be changed.

Sing-Li commented on src/lib/components/docslink/Docslink.svelte:
```diff
@@ -12,15 +12,36 @@
 </script>
 <style>
-    p { text-align: center; }
+    .links-container {
```
Needs to be a little bigger to balance.  

<img width="867" alt="Screenshot 2025-03-20 at 3 52 58 AM" src="https://github.com/user-attachments/assets/cff41c06-eaf0-4fd9-a774-83b859314d3e" />

Sing-Li commented on src/lib/components/carousel/Carousel.svelte:
```diff
@@ -18,35 +18,35 @@
  let slider = $state("");
```
The other PR  (Carousel component fix / extnsion) is going to overwrite this. 

Thanks for this work though.

Sing-Li reviewed (changes requested):
Overall, thanks for the excellent work @saaxena !    

Please make the requested changes ASAP.  As the other 2 PRs will depend on this one being merged first :pray:

saaxena commented:
I've implemented all the requested changes @Sing-Li . Please review and let me know if any further modifications are needed. 
Thanks! 🙌

Changes Made - 
Increased font size for better balance:
Changed from 0.95rem to 1.1rem to make the text more proportional with other components

Closed up the white space gap:
Modified the padding from padding: 1rem 0; to padding: 0.5rem 0 1rem;
This reduces the top padding while maintaining the bottom padding.

Sing-Li commented:
Actually, the white space above is still not fixed.   But it is better to merge it now and fix it later. @saaxena

saaxena commented:
Working on it. Thanks for review
